mlflow/server/js/src/common/utils/Utils.js

```diff
@@ -298,6 +317,17 @@ class Utils {
         sourceVersion +
         '/' +
         bitbucketMatch[3];
+    } else if (gitMatch) {
+      // eslint-disable-next-line no-unused-vars
+      const [_fullUrl, baseUrl, repoDir, fileDir] = gitMatch;
```

Comment:
```suggestion
      const [, baseUrl, repoDir, fileDir] = gitMatch;
```
@harupy
Thank you for review.
Maybe I should change this too?

Reply:
Thanks for the fix!